sycl/source/detail/kernel_bundle_impl.hpp

```diff
@@ -192,7 +192,6 @@ class kernel_bundle_impl {
     // TODO: Unify with c'tor for sycl::comile and sycl::build by calling
```

Comment:
Can we add a test to sycl/unittests/SYCL2020 ?

Reply:
I have a lit test for the multiple kernel stuff here: https://github.com/intel/llvm-test-suite/pull/440
Is that what you are asking?

Reply:
No, I was asking about adding a unit/component test in `sycl/unittests/SYCL2020` directory of this repo. This test should be able to validate this behavior without any backend available.
Let's do it in a separate PR?